v3/tutorials/07-add-a-pallet/index.mdx

```diff
@@ -492,14 +505,15 @@ Now you are ready to compile and run your node that has been enhanced with nickn
 from the [Nicks pallet](/rustdocs/latest/pallet_nicks/index.html). 
 
 ### Compile and run the node
+
 Compile the node in release mode with:
 
 ```bash
 cargo build --release
 ```
 
 If the build fails, go back to the previous section and make sure you followed all the steps
-correctly. You can also view a [full solution including the `Nick` pallet](https://github.com/substrate-developer-hub/substrate-node-template/tree/tutorials/solutions%2Fadd-a-pallet-v3.0.0+monthly-2021-05).
+correctly. You can also view a [full solution including the `Nick` pallet](https://github.com/substrate-developer-hub/substrate-node-template/tree/tutorials/solutions%2Fadd-a-pallet-v3).
```

Comment:
Looks like this is tested with latest monthly?

Reply:
I merge `monthlyly-2021-10` of node template into add a pallet. Refer to:
https://github.com/substrate-developer-hub/substrate-node-template/commits/tutorials/solutions/add-a-pallet-v3

This afternoon I pushed two tags `tut-sols/add-a-pallet/latest` and `tut-sols/add-a-pallet/monthly-2021-10` in node template. The pattern will be similar to the monthly tag and `latest` tag used on the main branch.